openesdh-repo/src/main/java/dk/openesdh/repo/webscripts/authorities/CaseAuthoritiesWebScript.java

```diff
@@ -0,0 +1,174 @@
+
+import com.github.dynamicextensionsalfresco.webscripts.annotations.HttpMethod;
+import com.github.dynamicextensionsalfresco.webscripts.annotations.Uri;
+import com.github.dynamicextensionsalfresco.webscripts.annotations.WebScript;
+import com.github.dynamicextensionsalfresco.webscripts.resolutions.Resolution;
+import com.google.common.base.Joiner;
+import dk.openesdh.repo.model.OpenESDHModel;
+import static dk.openesdh.repo.webscripts.ParamUtils.getRequiredTemplateParam;
+import dk.openesdh.repo.webscripts.utils.WebScriptUtils;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.alfresco.model.ContentModel;
+import org.alfresco.query.PagingRequest;
+import org.alfresco.query.PagingResults;
+import org.alfresco.repo.security.authority.AuthorityInfo;
+import org.alfresco.service.cmr.repository.InvalidNodeRefException;
+import org.alfresco.service.cmr.repository.NodeRef;
+import org.alfresco.service.cmr.repository.NodeService;
+import org.alfresco.service.cmr.security.AuthenticationService;
+import org.alfresco.service.cmr.security.AuthorityService;
+import org.alfresco.service.cmr.security.AuthorityType;
+import org.alfresco.service.cmr.security.PersonService;
+import org.alfresco.service.namespace.QName;
+import org.alfresco.util.Pair;
+import org.apache.commons.lang3.StringUtils;
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.extensions.webscripts.WebScriptException;
+import org.springframework.extensions.webscripts.WebScriptRequest;
+import org.springframework.extensions.webscripts.WebScriptResponse;
+import org.springframework.stereotype.Component;
+
+@Component
+@WebScript(description = "Get groups/users for case by case type", families = {"Authorities"})
+public class CaseAuthoritiesWebScript {
+
+    private static final String CREATED_ON_OPEN_E = "OPENE";
+    private static final int MAX_ITEMS = 10000;
+    private static final PagingRequest PAGING_REQUEST = new PagingRequest(MAX_ITEMS);
+    private static final List SORT_PROPS = Arrays.asList(
+            new Pair(ContentModel.PROP_FIRSTNAME, true),
+            new Pair(ContentModel.PROP_LASTNAME, true)
+    );
+    private static final List<QName> FILTER_PROPS = Arrays.asList(
+            ContentModel.PROP_FIRSTNAME,
+            ContentModel.PROP_LASTNAME);
+
+    @Autowired
+    private AuthorityService authorityService;
+    @Autowired
+    private NodeService nodeService;
+    @Autowired
+    private AuthenticationService authenticationService;
+    @Autowired
+    private PersonService personService;
+
+    @Uri(value = "/api/openesdh/{caseType}/authorities", method = HttpMethod.GET, defaultFormat = "json")
+    public Resolution getCaseAuthorities(WebScriptRequest req, WebScriptResponse res) throws JSONException {
+        String caseType = getRequiredTemplateParam(req, "caseType");
```

Comment:
I guess the @UriVariable annotation would apt here. Look at the dk.openesdh.repo.webscripts.workflows.WorkflowTaskDetailsWebScript as a sample.